index.html

```diff
@@ -178,5 +178,29 @@ <h1 class="upper_word"><br>About me</h1>
                 </ul>
             </div>
     </section>
+
+    <section id="contact-us">
+        <h1 class="upper_word">
+            I'm always interested in hearing about new projects, 
+            so if you'd like to chat please get in touch.
+        </h1>
+        
+            <form action="https://formspree.io/f/xyyargbq"  method="POST" class="flex-contact">
+
+                <input type="text" name="name" placeholder="Full name" class="flex-name" required>
+                <input type="email" name="email" placeholder="Email address" class="flex-email" required>
+                <textarea name="message" placeholder="Hey I’d like to connect to..." class="flex-message" required></textarea>
+                <button type="submit" class="flex-button">Get in touch</button>
+            </form>
```

Comment:
- [ ] Impressive work in implementing the project requirements but unfortunately I can't approve your project because you missed parts of the requirements. 
   - As specified in the project requirements kindly make sure to add a `maxLength` on the name and message inputs such that users are unable to input too much information than needed. See below for more details:
   
<img width="523" alt="Screenshot 2023-06-13 at 12 59 04" src="https://github.com/ishimwepati/Portfolio/assets/107192076/ece40089-8d25-423f-ab4c-5f954402fbb1">

Reply:
Thanks a lot fot the suggestion!